src/components/Todo/index.js

```diff
@@ -41,8 +41,12 @@ export default class Todo extends React.Component {
           <DropdownItem onClick={() => onDelete(todo.id)}>
             <span className="has-text-danger">Delete</span>
           </DropdownItem>
+          <DropdownDivider/>
+            <DropdownItem onClick={() => onSnooze(todo, 1, console.log(todo.newTodoDate))}>Snooze 1 day</DropdownItem>
```

Comment:
The `console.log(todo.newTodoDate)` is very weird and very incorrect. It is not a parameter to the function ? Also, `todo.newTodoDate` is not defined ? Particularly before sending the action, `todo.dueDate` has not been changed yet.

What did you want to do with this log ? Just to be sure, what I meant in the google doc was that you could start simply with:
```jsx
<DropdownItem onClick={() => console.log('Snooze 1 day')}>Snooze 1 day</DropdownItem>
<DropdownItem onClick={() => console.log('Snooze 3 days')}>Snooze 3 days</DropdownItem>
<DropdownItem onClick={() => console.log('Snooze 7 days')}>Snooze 1 week</DropdownItem>
<DropdownItem onClick={() => console.log('Snooze 21 days')}>Snooze 3 weeks</DropdownItem>
```

Reply:
mdrr nan c'est un truc que j'ai oublié d'enlever avant de push connuuu

Reply:
j'avais bien fait les console.log au début, c'est juste la j'essayais de console log qqch d'autre pour voir pk ça modifiait pas la date mais celui la en l'occurence j'ai juste oublié de supprimer ma catchatte haha